subsys/nrf_security/src/legacy/CMakeLists.txt

```diff
@@ -165,3 +177,23 @@ target_include_directories(${mbedcrypto_target}
 INTERFACE
   $<$<TARGET_EXISTS:nrf_cc3xx_platform>:$<TARGET_PROPERTY:nrf_cc3xx_platform,INTERFACE_INCLUDE_DIRECTORIES>>
 )
+
+if(NOT is_getting_includes)
+  if(NOT DEFINED ARM_MBEDTLS_PATH)
+	message(FATAL_ERROR "misconfiguration")
```

Comment:
Should be 2 space indentation, not tabs

Same on target_include_directories further down